File: 2706. Buy Two Chocolates.py
class Solution:
    def buyChoco(self, prices: List[int], money: int) -> int:
        min1 = min2 = float('inf')

        for price in prices:
            if price < min1:
                min1, min2 = price, min1
            elif price < min2:
                min2 = price

        if min1 + min2 <= money:
            return money - (min1 + min2)
        return money

File: 2706. Buy Two Chocolates.cpp
class Solution {
public:
    int buyChoco(vector<int>& prices, int money) {
        int min1 = 101;
        int min2 = 101;

        for (int &price : prices) {
            if (price < min1) {
                min2 = min1;
                min1 = price;
            } else if (price < min2) {
                min2 = price;
            };
        };

        if (min1 + min2 <= money) {
            return money - (min1 + min2);
        };
        return money;
    };
};
